templates/timescale.yaml

```diff
@@ -7,6 +7,7 @@ metadata:
   namespace: {{ .Release.Namespace }}
   labels:
     app: timescale
+    coder.deployment: postgresql
```

Comment:
I called this postgresql because, well, the pod is actually postgresql... but we can also call it timescale if y'all think that's better

Reply:
We should call it timescale to match the app label and the deployment name

Reply:
So we're gonna stick with this lie forever? :rofl: Fair enough I guess